routes/campaigns/search.php

```diff
@@ -43,3 +44,5 @@
 Route::get('/w/{campaign}/search/recent', [RecentController::class, 'index'])->name('search.recent');
 
 Route::get('/w/{campaign}/search/fulltext', [FullTextController::class, 'index'])->name('search.fulltext');
+
+Route::get('/w/{campaign}/search/json', [EntityJsonController::class, 'jsonSearch'])->name('search.json');
```

Comment:
Bind the entity in the url. Pass 0 in the setup to bypass laravel complaining and replace it with js